geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/audit/AccumuloEventWriter.scala

```diff
@@ -49,10 +49,10 @@ class AccumuloEventWriter(connector: AccumuloClient, table: String) extends Runn
     var toMutation = queue.poll()
     if (toMutation != null) {
       val writer = getWriter
-      do {
+      while (toMutation != null && running.get) {
```

Comment:
Is there ever a case where `running.get` can be false so the mutation sin't added?

Reply:
yeah, that's by design. it only runs every 5 seconds, and will drop anything that was queued up when the system shuts down or the datastore is disposed. might be worth reconsidering, but i think the behavior hasn't changed in this PR.